Source/Scene/Cesium3DTileset.js

```diff
@@ -2804,6 +2808,34 @@ Cesium3DTileset.prototype.destroy = function () {
   return destroyObject(this);
 };
 
+Cesium3DTileset.supportedExtensions = {
+  "3DTILES_metadata": true,
+  "3DTILES_implicit_tiling": true,
+  "3DTILES_content_gltf": true,
+  "3DTILES_multiple_contents": true,
+  "3DTILES_bounding_volume_S2": true,
+  "3DTILES_batch_table_hierarchy": true,
+  "3DTILES_draco_point_compression": true,
+};
+
+/**
+ * Checks to see if a given extension is supported by Cesium3DTileset. If 
+ * the extension is not supported by Cesium3DTileset, it throws a RuntimeError.
+ *
+ * @param {3DTILES_metadata} extensionsRequired The extensions we wish to check
+ *
+ * @private
+ */
+Cesium3DTileset.checkSupportedExtensions = function (extensionsRequired) {
+  for (var extension in extensionsRequired) {
+    if (extensionsRequired.hasOwnProperty(extension)) {
```

Comment:
:open_mouth: My bad, I had forgotten that `extensionsRequired` is an array, not an object. This should be changed to a `for (var i = 0; i < extensionsRequired.length; i++)` loop instead.

Reply:
Good catch!

Reply:
I think `ModelUtility.js` has the same mistake but works accidentally. @srothst1 could you open a PR to fix that as well?